compile.sh

```diff
@@ -23,6 +23,9 @@ cp -r Publish-Canvas out/
 # Copy Publish-Canvas
 cp -r Publish-Video out/
 
+# Copy Stereo-Audio
+cp -r Stereo-Audio out/
+
```

Comment:
You're going to need to add the sed step here too to update the config.js file.

`sed -i -- 's/http:\/\/YOUR-SERVER-URL/https:\/\/opentok-web-samples-backend.herokuapp.com/g' out/Stereo-Audio/js/config.js`

Reply:
Ah I missed that, thanks!